environment/src/main/java/com/sequenceiq/environment/environment/service/EnvironmentCreationService.java

```diff
@@ -136,6 +137,7 @@ private void validateCreation(EnvironmentCreationDto creationDto, Environment en
     }
```

Comment:
I have changed the error message to  "Provisioning in " + creationDto.getCloudPlatform() + " is not enabled for this account." locally.

Reply:
👍